.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Launch cli - up",
+            "program": "${workspaceFolder}/src/spec-node/devContainersSpecCLI.ts",
+            "cwd": "${workspaceFolder}",
+            "args": ["up", "--workspace-folder", "../features-playground", "--log-level", "debug", ],
```

Comment:
Perhaps we can add in a very simple feature set/collection as one of the tests, and use that here?  That way we can always F5 into something useful.